src/features/advanced_analytics_panel/components/AdvancedAnalyticsContainer/AdvancedAnalyticsContainer.tsx

```diff
@@ -10,6 +9,7 @@ import { focusedGeometryAtom } from '~core/shared_state';
 import { advancedAnalyticsResourceAtom } from '~features/advanced_analytics_panel/atoms/advancedAnalyticsResource';
 import { AdvancedAnalyticsEmptyState } from '~features/advanced_analytics_panel/components/AdvancedAnalyticsEmptyState/AdvancedAnalyticsEmptyState';
 import { AdvancedAnalyticsDataList } from '~features/advanced_analytics_panel/components/AdvancedAnalyticsDataList/AdvancedAnalyticsDataList';
+import { i18n } from '~core/localization';
```

Comment:
as we've merged translation tokens, please add new one to 
src/core/localization/translations/en/common-messages.json